package.json

```diff
@@ -28,9 +28,7 @@
   "keywords": [
     "homebridge-plugin"
   ],
-  "dependencies": {
-    "typed-rest-client": "^1.7.3"
-  },
+  "dependencies": {},
```

Comment:
This may be just me being naive re dependencies but isn't typed-rest-client a runtime dependency, not just dev? Please correct me if I'm wrong here!

Reply:
You could be right, the way I understood it is that it would get bundled into the build.

Reply:
Yeah, I think we might want this back under `dependencies`: https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c

Sort that out and I'll be happy to approve